Files/MultilingualResources/Files.de-DE.xlf

```diff
@@ -339,23 +339,23 @@
           <target state="translated">Dieser Ordner ist leer.</target>
         </trans-unit>
         <trans-unit id="NavBackButton.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
-          <source>Back (Alt+Left Arrow)</source>
+          <source>Back (Alt + Left Arrow)</source>
```

Comment:
Any reason for this change?

Reply:
This change in particular or all the shortcut related ones?

A space between the + looks nicer especially for shortcuts like Alt+Strg++ → Alt + Strg + +

Reply:
I think it's best to keep it without the space in order to stay consistent.